jumpcloud-ADMU/Powershell/Public/Start-Migration.ps1

```diff
@@ -241,37 +241,43 @@ Function Start-Migration {
                 # If not $isForm, validate the API Key and OrgID
                 if (!$isForm) {
                     # Get the org from the API KEY
+                    try {
+                        $OrgSelection, $MTPAdmin = (Get-MtpOrganization -apiKey $JumpCloudAPIKey -orgId $JumpCloudOrgID)
+                    } catch {
+                        Throw [System.Management.Automation.ValidationMetadataException] "Provided JumpCloudAPIKey and OrgID could not be validated"
+                    }
                     $OrgSelection, $MTPAdmin = (Get-MtpOrganization -apiKey $JumpCloudAPIKey -orgId $JumpCloudOrgID)
                     # set the orgID and orgName
                     $ValidatedJumpCloudOrgName = "$($OrgSelection[1])"
                     $ValidatedJumpCloudOrgID = "$($OrgSelection[0])"
-                    If (-Not $ValidatedJumpCloudOrgID) {
+                    If ([string]::IsNullOrEmpty($ValidatedJumpCloudOrgID)) {
```

Comment:
Previously the `Get-MTPOrganization` function was returning something for orgSelection and somehow that wasn't being evaluated correctly here. This is a more explicit test for whether or not some organization ID is returned.